system/system_error_monitor/config/diagnostic_aggregator/localization.param.yaml

```diff
@@ -14,6 +14,12 @@
               contains: [": localization_topic_status"]
               timeout: 1.0
 
+            tf_status:
+              type: diagnostic_aggregator/GenericAnalyzer
+              path: tf_status
+              contains: [": localization_tf_status"]
+              timeout: 1.0
+
```

Comment:
Actually, I've removed this since we don't use this anymore. There is the `pose(vehicle_kinematic_state)` topic instead.
What is your motivation for adding it?

Reply:
> What is your motivation for adding it?

Now, ad_service_state_monitor publishes it, so I'm sure it is used in autoware.

> There is the pose(vehicle_kinematic_state) topic instead.

Could you tell me where it is written?

Reply:
Here.
https://github.com/autowarefoundation/autoware.universe/blob/d9460e5040777899a614fe4ddbfb88f251b2cb69/system/ad_service_state_monitor/config/ad_service_state_monitor.param.yaml#L68-L72

Reply:
Thank you for your information. I confirmed it.
So, as next action, two choices are suggested.
- delete tf diag since this is not used.
- merge this PR to check tf. Because tf and topic are different.

Do you have any ideas?

Reply:
I think you can delete this diag, but before that, I think we should discuss the use case and consider what to check.

Reply:
I got it. I'll make issue to discuss it. I'll change to draft.

Reply:
I made https://github.com/autowarefoundation/autoware/discussions/107